content/blog/2021/07/camel-k-release-1-5/index.md

```diff
@@ -0,0 +1,61 @@
+
+You can learn more in [a blog dedicated to the topic](/blog/2021/05/new-camel-k-logging-features/).
+
+## S3 dependencies
+
+**S3** protocol is often used as a way to store Maven dependencies. With version 1.5 you will be able to configure your Camel K to retrieve such dependencies.
+
+## Configuration refactoring
+
+Prior to this version, we may have found a bit clumsy the work of providing a configuration file to our `Integration`. As a developer, you will now be able to define in details how and where to load a configuration file, filter a `Configmap` or a `Secret` and also distinguish between __build-time__ and __runtime__ properties. An entire blog covering this feature will be soon available, stay tuned!
+ 
+## More Kamelet commands
+
+Since their appearence, **Kamelets** have gained quite a lot of traction. With this new release we've introduced new commands in the `kamel` CLI to help you discovering and managing them. You will have a new command available `kamel kamelet` that will help you listing and deleting any `Kamelet`. Also `kamel describe kamelet` has been introduced to help you analyising the available Kamelets.
+
+## Allow configuration via traits annotations
+
+We are aware that the CLI is not the only way to create an `Integration`. For this reason we've introduced a new way to simplify the setting of a `trait` directly in the `Integration` specification. Instead of specifying a trait in the `.spec.integration.traits.trait-id.configuration.trait-property` you will be now able to do so via `.metadata.annotations`.
+
+## More test coverage
+
+Of course, we haven't forgot to include more unit and integration tests to our code base. With this release we've introduced new test cases that are giving more confidence to the development lifecycle.
+
+## Polish documentation
+
+We are happy to announce a few new improvements in the online documentation. We're dedicating some more time to refresh the documentation that may be missing or that can be incomplete.
+
+### Versioned documentation
+
+One little but great improvements we've introduced lately is the versioning of the Camel K documentation website. Just look on the left bottom bar of your browser and you will be able to select either `latest` or `1.4.x`.
+
+### Generated API
+
+Another thing that is worth to mention is a first version of our API that is now available through the documentation pages. Have a look at [Camel K API resources documentation page](/camel-k/latest/apis/camel.html) in order to understand how to interact directly with them.
+
+# Thanks
+
+Thanks to all contributors who made this possible. We're happy to receive feedback on this version through the [usual channels](/camel-k/latest/contributing/developers.html) or filing an issue on [Camel K Github repository](https://github.com/apache/camel-k).
```

Comment:
The https://camel.apache.org/camel-k/latest/contributing/developers.html page is more about how to setup the environment, not really how to provide feedback. But maybe this article should stay like that and the page contriguting shoudl include information on mailing list, chat and bug tracker?

Reply:
Yeah, I think we do miss a landing page for feedback. Just created an [issue](https://github.com/apache/camel-k/issues/2502). I'll rewrite that differently in the while.